inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/HttpServletRequestFilter.java

```diff
@@ -38,9 +35,6 @@
  * Make All
  */
 @Slf4j
-@Component
-@WebFilter
-@Order(0)
```

Comment:
Why remove those annotations?

Reply:
All web filter will be managed in WebFilterConfig